### PR TITLE
OAuth additions in docs (scope was missing)

### PR DESCRIPTION
**Pull request by bluedeepimpact**
https://community.openproject.org/work_packages/36589

### CODE REVIEW

**Comment by bluedeepimpact** [comment]
@wielinde may I ask you to add the technical info here (where the "...." are in the text)? Thank you in advance!

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 1]
```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
...
+To activate and configure OAuth applications, navigate to -> *Administration* -> *Authentication* -> *OAuth applications*.
 
-## Add a new authentication application for oauth
+## Add a new authentication application for OAuth
 
-To add a new oauth application, click the green **+ Add** button.
+To add a new OAuth application, click the green **+ Add** button.
 
-![Sys-admin-authenticatoin-oauth-applications](Sys-admin-authenticatoin-oauth-applications.png)
+![Sys-admin-authenticatoin-OAuth-applications](Sys-admin-authenticatoin-OAuth-applications.png)
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: Careful when rigorously search and replace "oauth" with "OAuth". The file path here is wrong now.

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 2]
```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
...
-3. Check if the application will be used **confidentially**.
-4. (Optional) Choose **client credential flows** and define a user on whose behalf requests will be performed.
-5. Press the blue **Create** button to add your oauth application.
+3. .................................
+4. Check if the application will be used **confidentially**.
+5. (Optional) Choose **client credential flows** and define a user on whose behalf requests will be performed.
+6. Press the blue **Create** button to add your OAuth application.
 
-![Sys-admin-authentication-add-oauth-application](Sys-admin-authentication-add-oauth-application.png)
+![add-new-oauth-application](image-20210315182306678.png)
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: That file path is not human readable. The file name does not say what that PNG is about. It is better practice to name files in a way that they are easy to understand.

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 3]
```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
...
 Don't forget to note down your `Client ID` and your `Client secret` in a safe space. You will need it later.
 
-## Oauth endpoints
+## OAuth endpoints
 
 The authentication endpoints are at
 
-* Auth URL: `https://example.com/oauth/authorize`
-* Access Token URL: `https://example.com/oauth/token`
+* Auth URL: `https://example.com/OAuth/authorize`
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: don't replace it here. It's bad practice to use capital letters in URIs

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 4]
```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
...
+* Access Token URL: `https://example.com/OAuth/token`
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: don't replace it here. It's bad practice to use capital letters in URIs

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 5]
```diff
@@ -48,7 +49,7 @@ Request an authorization code. Please adopt the following URL replacing
  * `<Redirect URI>` with the redirect URI just as you configured it above.
  * You can leave the `scope` value untouched unless you are running the OpenProject BIM edition and also plan to access to the BCF version 2.1 REST API. Then simply replace `api_v3` with `api_v3%20bcf_v2_1`.
 
-`https://example.com/oauth/authorize?response_type=code&client_id=<Client ID>&redirect_uri=<Redirect URI>&scope=api_v3&prompt=consent`
+`https://example.com/OAuth/authorize?response_type=code&client_id=<Client ID>&redirect_uri=<Redirect URI>&scope=api_v3&prompt=consent`
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: don't replace it here. It's bad practice to use capital letters in URIs

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 6]
```diff
@@ -62,15 +63,15 @@ In this example we skip that server side implementation and just copy the value
...
+
  * the `example.com` with the root path of your OpenProject instance,
  * `<Client ID>` with your client ID
  * `<Authentication code>` with the code you obtained above,
  * and `<Redirect URI>` with the redirect URI just as you configured it above.
 
 ```
 $ curl --request POST \
-  --url 'https://example.com/oauth/token' \
+  --url 'https://example.com/OAuth/token' \
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: don't replace it here. It's bad practice to use capital letters in URIs

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 7]
```diff
@@ -94,15 +95,15 @@ For example, the following cURL command fetches all projects from the API V3. Pl
 `$ curl --request GET 'https://example.com/api/v3/projects' \
   --header 'Authorization: Bearer <Token>'`
 
-## Using Postman with oauth?
+## Using Postman with OAuth?
 
-Set redirect URLs to `urn:ietf:wg:oauth:2.0:oob` in both, for your application (see step 2 above) and 
+Set redirect URLs to `urn:ietf:wg:OAuth:2.0:oob` in both, for your application (see step 2 above) and
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: oh oh, please don't replace it here.

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 8]
```diff
@@ -94,15 +95,15 @@ For example, the following cURL command fetches all projects from the API V3. Pl
...
 
-Set redirect URLs to `urn:ietf:wg:oauth:2.0:oob` in both, for your application (see step 2 above) and 
+Set redirect URLs to `urn:ietf:wg:OAuth:2.0:oob` in both, for your application (see step 2 above) and 
 within Postman.
 
 In Postman the configuration should look like this (Replace `{{protocolHostPort}}` with your host, 
 i.e. `https://example.com`)
 
-![Sys-admin-authentication-add-oauth-application](Sys-admin-authentication-oauth-postman.png)
+![Sys-admin-authentication-add-OAuth-application](Sys-admin-authentication-OAuth-postman.png)
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: don't replace it here.

File: docs/system-admin-guide/authentication/oauth-applications/README.md [hunk 9]
```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
...
-You can configure the following options to add your oauth application.
+You can configure the following options to add your OAuth application.
 
-1. Enter the **name** of your oauth application.
+1. Enter the **name** of your OAuth application.
 2. **Define redirect URLs** where authorized users can be redirected to.
-3. Check if the application will be used **confidentially**.
-4. (Optional) Choose **client credential flows** and define a user on whose behalf requests will be performed.
-5. Press the blue **Create** button to add your oauth application.
+3. .................................
```
Comment (wielinde) [inline docs/system-admin-guide/authentication/oauth-applications/README.md]: Here my 2 cents:

Set the **scope** that the OAuth application will have access to. Chose the API you want to grant access to. Multiple selections are possible. If no scope is checked, per default **api_v3** is assumed.

**Review by wielinde — CHANGES_REQUESTED** [review]
Hi @ML-OpenP, please find my review attached. I also filled in the one .... gap. I hope I didn't miss any other gap. 

One very important thing. Be careful with search and replace. Always review every replacement. Pay extra attention to file names and URIs.

**Pull request merged**